subsys/partition_manager/Kconfig

```diff
@@ -141,6 +141,7 @@ if BUILD_WITH_TFM
 
 config PM_PARTITION_SIZE_TFM_RAM
 	hex "Memory reserved for TFM_RAM."
+	default 0x10000 if NRF_MODEM_LIB
```

Comment:
Should we create a task to update this once https://github.com/nrfconnect/sdk-nrf/pull/3792 is merged? I guess we should update this once the amount of RAM used by modem lib is reduced.

Reply:
Good idea. Will do

Reply:
Done: NCSDK-8184